misc/scripts/bwrap.sh

```diff
@@ -27,7 +27,7 @@ function safe_source() {
     mkdir "${PACDIR}" 2> /dev/null
     tmpfile="$(sudo mktemp -p "${PACDIR}")"
     local allvar_str pacfunc_str debfunc_str pacstall_funcs=("prepare" "build" "check" "package") \
-        debian_funcs=("post_install" "post_remove" "post_upgrade" "pre_install" "pre_remove" "pre_upgrade")
+    debian_funcs=("post_install" "post_remove" "post_upgrade" "pre_install" "pre_remove" "pre_upgrade")
```

Comment:
Don't remove the indentation, it's a line break

Reply:
for note purposes, we discussed in the discord that shfmt insisted on this